pkg/api/authn.go

```diff
@@ -197,52 +181,21 @@ func basicAuthHandler(ctlr *Controller) mux.MiddlewareFunc {
 
 				return
 			}
-			if (request.Method == http.MethodGet || request.Method == http.MethodHead) && ctlr.Config.HTTP.AllowReadAccess {
+			if request.Header.Get("Authorization") == "" && checkAnonymousPolicyExists(ctlr.Config.AccessControl) {
```

Comment:
seems like we can't rely on this check because tools are sending this header when no user or password was provided:
Authorization: Basic Og==  (decoded value == ":")

The good news is that our function won't throw an error for this header value.

so we can add this check after line 191: username, passphrase, err := getUsernamePasswordBasicAuth(request)

```
			if username == "" && passphrase == "" && checkAnonymousPolicyExists(ctlr.Config.AccessControl) {
				// Process request
				next.ServeHTTP(response, request)

				return
			}
```